packages/api-client/src/client/ClientAPI.ts

```diff
@@ -125,8 +135,9 @@ export class ClientAPI {
 
   /**
    * Will upload keypackages for an MLS capable client
-   * see https://staging-nginz-https.zinfra.io/api/swagger-ui/#/default/post_mls_key_packages_self__client_
-   * @param  {string} clientId
+   * @see https://staging-nginz-https.zinfra.io/api/swagger-ui/#/default/post_mls_key_packages_self__client_
+   * @param  {string} clientId The client to upload the key packages for
+   * @param  {Array.<string>} keyPackages The key packages to upload
```

Comment:
```suggestion
   * @param {string} clientId The client to upload the key packages for
   * @param {string[]} keyPackages The key packages to upload
```

I am curious where this `Array.<string>` comes from, is the "." a typo?

Reply:
its another option for typing in jsdoc, but I agree that synchronising with typescript types is preferable
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#types-1